ConsoleBank/ConsoleBank/Currencies/CurrencyInfo/ICurrencyInfo.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Currencies
+{
+    public interface ICurrencyInfo
+    {
+        Task<CurrencyRate> GetCurrenciesRateOnDate(string currencyAbbreviation, DateTime date);
+        Task<CurrencyRate> GetCurrenciesRate(string currencyAbbreviation);
```

Comment:
опциональный параметр?
плюс мы курс для одной валюты возвращаем - поэтому в одиночном числе
```suggestion
        Task<CurrencyRate> GetCurrencyRate(string currencyAbbreviation, DateTime? date);
```